api/models.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import unicode_literals
 
 import json
+from datetime import datetime
```

Comment:
Because timezone support is activated timezone aware datetime objects must be used. Please see [Django's Time zones docs](https://docs.djangoproject.com/en/1.8/topics/i18n/timezones/) for more information.